parent/pom.xml

```
@@ -18,7 +18,7 @@
   </description>
 
   <properties>
-    <version.spring.framework>5.2.8.RELEASE</version.spring.framework>
+    <version.spring.framework>5.2.13.RELEASE</version.spring.framework>
```

We could also go for `5.2.14.RELEASE` already, if we like.

```suggestion
    <version.spring.framework>5.2.14.RELEASE</version.spring.framework>
```